concourse/pipelines/destroy-microbosh.yml

```diff
@@ -98,6 +98,23 @@ jobs:
           - get: bosh-secrets
           - get: bosh-init-state
           - get: bosh-manifest
+      - task: bosh-cleanup
+        config:
+          platform: linux
+          image: docker:///governmentpaas/bosh-cli
+          inputs:
+            - name: paas-cf
+            - name: bosh-secrets
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                ./paas-cf/concourse/scripts/bosh_login.sh bosh-secrets/bosh-secrets.yml
+
+                # Cleanup ensures any orphaned disks are deleted before bosh itself is deleted
```

Comment:
That's good, because it's stuff that we also want to happen. But it means the comment is factually inaccurate and I don't think it adds much (compared to looking at the commit history to see exactly why we added it).